.travis.yml

```diff
@@ -4,17 +4,77 @@ os:
 
 dist: trusty
 
-language : c
+language: c
 
 compiler:
   - clang
   - gcc
+matrix:
+  include:
+    - compiler: "gcc"
+      os: linux
+      env:
+        - PLATFORM=linux BITS=64 HOST=x86_64
+        # TODO do `check` or `check_hw` here too, not just `check_sw` (as part of `coverage`)
+        - CHECK_RULE=coverage GCOV=1
+        - PKG_RULE=gzip
+        - COVERITY_SCAN_PROJECT_NAME="$TRAVIS_REPO_SLUG"
+        - COVERITY_SCAN_BRANCH_PATTERN="travis-ci"
+        - COVERITY_SCAN_NOTIFICATION_EMAIL="robin.getz@analog.com"
```

Comment:
No, it's an email every time it runs with the results. (Did you still want to be there?)